ui/v2.5/src/components/Settings/SettingsConfigurationPanel.tsx

```diff
@@ -328,10 +328,10 @@ export const SettingsConfigurationPanel: React.FC = () => {
 
   return (
     <>
-      <h4>Library</h4>
+      <h4>My Library</h4>
```

Comment:
I know why you are saying this but in terms of user experience it resonates more with a brain to say 'my library' in terms of knowing what it means vs "library" which is slightly vague. the "my" also implies ownership.  Just thinking in terms of end-user friendliness here.

Reply:
This would resonate more I think if such language was used elsewhere in the app, but it is not. I don't think it makes the intent of the section clearer by having `My` in it, and clashes with the language used elsewhere.